### PR TITLE
AF-2253: Tests improved for fs changes to default

### PR DESCRIPTION
**Pull request by adrielparedes**
## Intention

Replace all the references to JGit filesystem using Default Schema. That way other file system providers can be used.

## Related to

* https://github.com/kiegroup/appformer/pull/880
* https://github.com/kiegroup/kie-wb-common/pull/3289

### CODE REVIEW

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by rhtevan** [comment]
@adrielparedes @ederign LGTM, I'm fine with merge it.

File: kie-wb-common-services/kie-wb-common-services-backend/src/test/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
[the remark this answers is not in the document]
Comment (adrielparedes) [reply kie-wb-common-services/kie-wb-common-services-backend/src/test/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider]: done!

**Pull request merged**